### ISSUE TITLE
Update next link on "Data tests" page to "Unit tests" page 

### ISSUE THREAD

**Issue by sdopoku** [fishtown-analytics/docs.getdbt.com#5542]
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

https://docs.getdbt.com/docs/build/data-tests

### What part(s) of the page would you like to see updated?

The link to the next page at the bottom of this page currently redirects to the same page. It should, however, be linked to the Unit tests page: https://docs.getdbt.com/docs/build/unit-tests. 

I tried the "Edit this page" feature but could not locate the code to fix this myself. 😢  See the screenshot below:

<img width="841" alt="Screenshot 2024-05-22 at 12 45 46" src="https://github.com/dbt-labs/docs.getdbt.com/assets/4030902/c38839f6-ac4a-4c02-84c2-0bc7a2550824">

**Comment by mirnawong1** [comment]
great flag @sdopoku ! i've created a quick pr here to fix this: https://github.com/dbt-labs/docs.getdbt.com/compare/current...mirnawong1-patch-28?quick_pull=1

tl;dr, we needed to add a `pagination_next: "intended-link-here"` in the frontmatter.
